### ISSUE TITLE
Jobs should display whether or not they are suspended.

### ISSUE THREAD

**Issue by kannon92** [kubernetes/kubernetes#123221]
### What would you like to be added?

Suspend is a GA feature and it is big component of Kueue.

It would be nice to display if jobs are suspended when doing `kubectl get job`.

```
kehannon@kehannon-thinkpadp1gen4i:~/Work/jobset/examples/startup-policy$ kubectl get jobs
NAME                                        COMPLETIONS   DURATION   AGE
pytorch-job                                 0/4           8d         8d
pytorch-workers-0                           1/4           8d         8d
success-policy-workers-0                    2/2           14s        8d
suspend-startup-driver-anyorder-driver-0    0/1                      10s
suspend-startup-driver-anyorder-workers-0   0/4                      10s
suspend-startup-driver-ready-driver-0       0/1                      49s
suspend-startup-driver-ready-workers-0      0/4                      49s
```
A new column that displays if the job is suspended would be helpful.

### Why is this needed?

Improve UX around suspend. Its a bit difficult to know why the job has not created pods unless you know about the suspend field.

**Comment by kannon92** [comment]
/cc @alculquicondor @mimowo 

Just something I noticed as I was testing some job in JobSet. It would be nice to see which jobs are suspended when doing `kubectl get job`. Hope its not a controversial change.

**Comment by mimowo** [comment]
cc @ivanvc who worked on https://github.com/kubernetes/kubernetes/pull/122618

**Comment by alculquicondor** [comment]
Not controversial at all IMO

/triage accepted

**Comment by ivanvc** [comment]
Any suggestions on where to place this column? I'm thinking between completions and durations. Or, should this be displayed only when the output is set to wide?

**Comment by alculquicondor** [comment]
I would put it in the default view as well. Probably before completions

**Comment by ivanvc** [comment]
Addressed in the PR: #123226
